src/handlers/createHandler.tsx

```diff
@@ -5,6 +5,8 @@ import {
   DeviceEventEmitter,
   EmitterSubscription,
 } from 'react-native';
+// @ts-ignore - its taken straight from RN
+import { customDirectEventTypes } from 'react-native/Libraries/Renderer/shims/ReactNativeViewConfigRegistry';
```

Comment:
Sorry for butting into your PR. From your talks with the Meta team, did they say this is the way to do it and there is no other way? I am facing a similar issue trying to send Fabric events in bridgeless mode. This looks very brittle. What happens to older versions where this doesn't exist? Or if they ever decide to change this import?

Reply:
Ouch 😅. Alright, I'll probably be a bit more careful then 😂. Fingers crossed